src/ripple/net/impl/SSLHTTPDownloader.cpp

```diff
@@ -317,6 +363,8 @@ SSLHTTPDownloader::fail(
             "exception: " << e.what();
     }
     complete(std::move(dstPath));
+
```

Comment:
Consider logging where the exception was caught.
eg.
```
JLOG(j_.error()) <<
    "exception " << e.what() << " in function " << __func__;
```

Reply:
Done